node/src/daemon/setup_reporter.rs

```diff
@@ -835,6 +836,18 @@ impl ValueRetriever for MinHops {
     fn value_name(&self) -> &'static str {
         "min-hops"
     }
+
+    fn computed_default(
+        &self,
+        _bootstrapper_config: &BootstrapperConfig,
+        persistent_config: &dyn PersistentConfiguration,
+        _db_password_opt: &Option<String>,
+    ) -> Option<(String, UiSetupResponseValueStatus)> {
+        match persistent_config.min_hops() {
+            Ok(min_hops) => Some((min_hops.to_string(), Configured)),
+            Err(e) => panic!("Database is corrupt: {:?}", e),
```

Comment:
Not sure if we ever want to make Daemon panic. Remember that it starts up during booting the hosting machine. It's not a process that you could see as one of your open windows. Killing it means the user will not know what's wrong with our software and also how he could make it run again. He'd have to reboot his machine which isn't friendly from our side. So I'd consider (maybe to log first; feel free to create a new short-life logger in  place for that purpose) and pick the default if this happens. If the problems repeat in the Node too, it will be its end but Daemon should probably be steadier.

Reply:
Done 👍